label_sync/labels.yaml

```diff
@@ -15,13 +15,13 @@ default:
       prowPlugin: approve
       addedBy: approvers
     - color: fef2c0
-      description: Indicates a cherrypick PR into a release branch has been approved by the release branch manager. # Consumed by the kubernetes/kubernetes cherrypick-queue.
-      name: cherrypick-approved
+      description: Indicates a cherry-pick PR into a release branch has been approved by the release branch manager. # Consumed by the kubernetes/kubernetes cherry-pick-queue.
+      name: cherry-pick-approved
```

Comment:
Please use the `previously` field so that label_sync migrates the old label to the new label.

Reply:
Done.